orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -217,6 +229,9 @@ def _export_model(self, *inputs, **kwargs):
         #       Model is not re-exported when the model parameters change. This can happen when the model is a stateful model,
         #       or the user explicitly changed model parameters after the onnx export.
 
+        if self._fast_path == True:
```

Comment:
Here we pay a function call to just return False. We could change `InferenceManager` and `TrainingManager`'s forward implementation to prevent `_export_model`

Reply:
I refactor the code a bit, now I don't need to call the function everytime. I won't touch InferenceManager for now.